src/pydata_sphinx_theme/bootstrap_html_translator.py

```diff
@@ -62,3 +81,12 @@ def visit_table(self, node):
 
         tag = self.starttag(node, "table", CLASS=" ".join(classes), **atts)
         self.body.append(tag)
+
+    def visit_literal_block(self, node: nodes.Element) -> None:
+        """overwrite the literal-block element to make them focusable"""
+
+        # add tabindex to the node
+        pst_atts = getattr(node, "pst_atts", {})
```

Comment:
so pst_atts are chunks of data we are adding to the object on our own in order to re-use them later? And I guess we can't just modify `atts` and `classes` directly?

Reply:
`classes` and `atts` are keyword arguments of the `starttag` method. They are not reachable without a strong override of the `visit` method (as I did in my first proposal). In this case the advantage is to carry the information within the node object allowing me to make very small changes to visit methods: 
```python 
def visit_xxx(self, node):

    node.pst-class = "toto tutu tata"

    super().visit_xxx(node)
```

Reply:
Ah wait so pst is not short for "pydata sphinx theme"? These are in-built sphinx things?

Reply:
you are very right "pst" stands for pydata-sphinx-theme which guarantee that no other lib should interfere with these values